test/extended/util/client.go

```diff
@@ -674,6 +675,10 @@ func (c *CLI) BuildClient() buildv1client.Interface {
 	return buildv1client.NewForConfigOrDie(c.UserConfig())
 }
 
+func (c *CLI) DnsRecordClient() dnsrecordclientset.Interface {
+	return dnsrecordclientset.NewForConfigOrDie(c.AdminConfig())
```

Comment:
The functions for Admin user are started from Line 697, please move this function to proper location. 
And please update the function name to `AdminIngressClient`.

Reply:
thanks i missed it is an admin command